ampersand-multifield-view.js

```diff
@@ -138,12 +181,36 @@ var MultiFieldView = View.extend({
     View.prototype.remove.apply(this, arguments);
   },
 
+  reset: function() {
+    this.setValue(this.startingValue, true);
+    this.fields.forEach(function(field) {
+      if(field.reset) {
+        field.reset();
+      }
+    });
+  },
+
+  clear: function() {
+    // This won't unset field values, but will disable validation
+    this.setValue({}, true);
+
+    this.fields.forEach(function(field) {
+      if(field.clear) {
+        field.clear();
```

Comment:
In the case the fields support `clear()`. A lot of ampersand's views do so, but it's not a requirement for a field to support....